test/unit/execCmd.test.ts

```diff
@@ -33,7 +40,9 @@ describe('execCmd (sync)', () => {
     const shellString = new ShellString(JSON.stringify(output));
     const execStub = stubMethod(sandbox, shelljs, 'exec').returns(shellString);
     execCmd(cmd);
-    expect(execStub.args[0][0]).to.equal(`${binPath} ${cmd}`);
+    expect(execStub.args[0][0]).to.include(`${binPath} ${cmd}`);
+    expect(execStub.args[0][0]).to.include('> command');
```

Comment:
@WillieRuemmele I know this works, but should these be changed to `1> command` for completeness?